src/Playwright/Transport/Channels/ArtifactChannel.cs

```diff
@@ -20,83 +20,71 @@
 * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 * SOFTWARE.
-*
-*
-* ------------------------------------------------------------------------------
-* <auto-generated>
-* This file is generated by generate_dotnet_channels.js, do not edit manually.
-* </auto-generated>
 */
 
-using System;
-using System.Collections.Generic;
 using System.Text.Json;
 using System.Threading.Tasks;
 using Microsoft.Playwright.Core;
 using Microsoft.Playwright.Helpers;
 
-#nullable enable
 namespace Microsoft.Playwright.Transport.Channels
 {
-    internal class ArtifactChannelImpl : Channel<Artifact>
+    internal class ArtifactChannel : Channel<Artifact>
     {
-        public ArtifactChannelImpl(string guid, Connection connection, Artifact owner) : base(guid, connection, owner)
+        public ArtifactChannel(string guid, Connection connection, Artifact owner) : base(guid, connection, owner)
         {
         }
 
-        internal virtual async Task<string?> PathAfterFinishedAsync()
+        internal async Task<string> PathAfterFinishedAsync()
```

Comment:
Why return type change?

Reply:
`#nullable enable` which we have enabled in our generated types, but by default everything is nullable in c#.